README.md

```diff
@@ -18,6 +18,26 @@ If you are considering making a PR, please take a look at the Waffle board to se
 
 The following steps walk through getting the application running. For contributing guidelines see [here](https://github.com/codebar/planner/blob/master/CONTRIBUTING.md).
 
+## Getting Started Through Docker
+
+You will need to have Docker installed, https://docker.com/
+
+The current Dockerfile and docker-compose were closely copied from the guide: https://docs.docker.com/compose/rails/
+
+1. Clone the project
+
+Run this to build a docker environemnt:
```

Comment:
Need to change the layout here because it currently renders like this:
![image](https://cloud.githubusercontent.com/assets/1155816/23257092/2ddeef1c-f9ba-11e6-8890-e80021889af2.png)
I recommend the following:
```
1. Clone the project.
2. Run `docker-compose build` to build a docker environment.
3. Run `docker-compose run web rake db:create db:migrate db:seed` to create, migrate and seed the main database
4. Run `docker-compose up` to start the app.
```